profile/README.md

```diff
@@ -1,3 +1,5 @@
+<img alt="Software Gardening organization logo" src="https://raw.githubusercontent.com/software-gardening/brand/main/logo/logo-text.png?raw=true" style="max-width:100%;" width="500">
+
```

Comment:
Does `style` here even work? I think gh doesn't really support this in their md, just width/height attributes.

Reply:
Test.

Max width:

<img alt="Software Gardening organization logo" src="https://raw.githubusercontent.com/software-gardening/brand/main/logo/logo-text.png?raw=true" style="max-width:100%;" width="500">

No max width:

<img alt="Software Gardening organization logo" src="https://raw.githubusercontent.com/software-gardening/brand/main/logo/logo-text.png?raw=true" width="500">

Reply:
Thanks great point! Updating.